io/net/virt-net/lpm.py

```diff
@@ -165,15 +181,27 @@ def rsct_service_start(self):
         '''
         try:
             for svc in ["rsct", "rsct_rm"]:
-                process.run('startsrc -g %s' % svc, shell=True, sudo=True)
+                cmd = 'startsrc -g %s' % svc
+                if not self.using_peer: process.run(cmd, shell=True, sudo=True)
+                if self.using_peer: output = self.peer_session.cmd(cmd)                        
```

Comment:
Make this into an if else statement; if using_peer peer_session else process.run()

Reply:
Don't worry about making it one line I thought it would look cleaner